rita_client/src/operator_update/mod.rs

```diff
@@ -481,13 +487,12 @@ fn update_payment_settings(
     payment.balance_warning_level = new_settings.warning.into();
     if let Some(new_chain) = new_settings.system_chain {
         if payment.system_chain != new_chain {
-            set_system_blockchain(new_chain, &mut payment);
+            set_system_blockchain(new_chain, payment);
         }
     }
     if let Some(new_chain) = new_settings.withdraw_chain {
         payment.withdraw_chain = new_chain;
     }
-    payment
 }
```

Comment:
Since this is no longer returning anything in this fn does this check for withdraw_chain need to be handled and set somewhere or can this be removed

Reply:
payment is a mutable reference so the check is in fact changing the variable passed into the function

this is why mutable references are generally a bad idea, people lose track of when they are being modified. So maybe I should change this back to a return based structure.